src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/mvn/mvn_kernel_bs_fs_yx_bsv32.cpp

```diff
@@ -49,6 +55,11 @@ bool MVNKernel_bs_fs_yx_bsv32::Validate(const Params& p, const optional_params&
         return false;
 
     auto params = static_cast<const mvn_params&>(p);
+    // To be choosed b_fs_yx_fsv16_imad i8/u8 kernel.
+    if ((params.inputs[0].GetDType() == Datatype::UINT8 || params.inputs[0].GetDType() == Datatype::INT8)
+        && params.inputs[0].GetLayout() == DataLayout::b_fs_yx_fsv16) {
+        return false;
+    }
```

Comment:
If it is supported, I think it would be better not to return false. Instead, we can let `kernel_selector` choose fsv16_imad kernel with `priority` information.